alkos/kernel/arch/x86_64/boot64.nasm

```diff
@@ -5,10 +5,19 @@
           ; GDT64
           extern GDT64.Data
 
+          ; vga code
+          extern terminal_initialize
```

Comment:
We don't have vga anymore in the latest code. Someone will have to merge itttttttt